MonoGame.Framework.Content.Pipeline/ContentBuildLogger.cs

```diff
@@ -173,7 +174,8 @@ public virtual void LogWarning(string helpLink, ContentIdentity contentIdentity,
     /// <param name="filename">Name of the file containing future messages.</param>
     public virtual void PushFile(string filename)
     {
-        Log(filename);
+        filename = filename.Sanitize();
+        Log(filename.Sanitize());
```

Comment:
This will have an effect on mgcb.

Also no need for double Sanitize.

Reply:
> This will have an effect on mgcb.

@harry-cpp 

What sort of effect?